frontend/dockerfile/docs/reference.md

```diff
@@ -2310,7 +2310,7 @@ Therefore, to avoid unintended operations in unknown directories, it's best prac
 ## ARG
 
 ```dockerfile
-ARG <name>[=<default value>]
+ARG <name>[=<default value>] ...
```

Comment:
Wondering if the more correct presentation of this would be (more clearly indicating that the first key/value is required, and more are optional);

```suggestion
ARG <name>[=<default value>] [<name>[=<default value>]...]
```


Similar to how (e.g.) we do for `docker stop` and `docker service scale`;

```console
docker stop --help

Usage:  docker stop [OPTIONS] CONTAINER [CONTAINER...]
```

```console
docker service scale --help

Usage:  docker service scale SERVICE=REPLICAS [SERVICE=REPLICAS...]
```

Reply:
Doh! Too late, just merged 😂

Reply:
I guess your suggestion is more correct but this was also the variant used in `ENV` and `LABEL`. So modifications should be done together then.